arcade/gui/examples/scroll_area.py

```diff
@@ -16,12 +16,13 @@
 from arcade import Window
 from arcade.gui import UIManager, UIWidget, Property, Surface, UIDummy, UIEvent, bind, \
     UIMouseDragEvent, UIMouseScrollEvent, UIMouseEvent, UIBoxLayout, UIFlatButton, UIInputText
+from arcade.types import Point
 
 
 class UIScrollArea(UIWidget):
-    scroll_x = Property(default=0)
-    scroll_y = Property(default=0)
-    canvas_size = Property(default=(300, 300))
+    scroll_x = Property[float](default=0)
```

Comment:
Also change the default to a float

Reply:
I can do that, but I recently learned that python typechecking standards allow `int` everywhere `float` is expected.  They call this the "numeric tower."

To be honest I'm not totally sure what issues occur when using `int` where a `float` is expected.  I know using `float` where `int` is expected can cause issues, not sure about the other way around.

https://peps.python.org/pep-0484/#the-numeric-tower